### DISCUSSION TITLE
Broker Cluster SA missing clusterrolebinding for strimzi-kafka-broker

### DISCUSSION THREAD

**Discussion by ryanfaircloth** [strimzi/strimzi-kafka-operator#7987] [General]
**Describe the bug**
When creating a cluster using the rack feature the broker nodes must be granted GET for nodes resource. 

**To Reproduce**
Create a Kafka Cluster with rack awareness 

**Expected behavior**
Operator should create a clusterrolebinding with the ref role of strimzi-kafka-broker and a subject of the broker service account

**Environment (please complete the following information):**
 - Strimzi version: 0.32.1
 - Installation method: helm
 - Kubernetes cluster: 1.24.1
 - Infrastructure:  Amazon EKS

**Comment by scholzj** [comment]
I do not think this is a general issue - but something related to your environment. Typically, some other tooling deleting the ClusterRoleBinding such as in #7933. Please have a look at that discussion to what to look for in the operator log or share the debug operator log so that we can confirm it.
